### ISSUE TITLE
Remove 'collection address' from order confirmation.

### ISSUE THREAD

**Issue by sstead** [openfoodfoundation/openfoodnetwork#636]
@kirstenalarsen @RohanM 

Hey Guys,
This has arisen as a problem for Susie in Launceston... she has 2 pickup points, as shipping methods.

![image](https://cloud.githubusercontent.com/assets/10843056/8922864/5313e4bc-352c-11e5-9739-cc9562062d88.png)

Is there any reason we couldn't just delete this 'collection address' field?Or change it to 'enterprise address'. We'd just need to make sure anyone using collection as a shipping method puts the pickup address in the shipping method description box.

**Comment by kirstenalarsen** [comment]
Thanks @sstead - can you check this out carefully with some other peoples' data, is there a reason why it hasn't come up before, and would the proposed solution mess anyone else up? Once you are really confident that you've got a 'generic' fix that covers all the different use cases, you can allocated this to @RohanM

**Comment by sstead** [comment]
Here's a breakdown of how the customer's order confirmation email is composed, depending on if they select a 'pick up' or 'delivery' shipping method...

![image](https://cloud.githubusercontent.com/assets/10843056/9000202/141ff7f2-3787-11e5-98ad-f4dea2c8fd20.png)

For pick up shipping methods... If the pick up address and enterprise address aren't the same there is definitely the possibility of conflicting info (customers get confused/go to the wrong place.)

Why hasn't this been an issue before? I think so far most enterprises have put plenty of detail in the shipping method name and description box, such as 'come to Warrigul farmers market between 10 and 2 to pick up your order'.... so the customer follows this instruction rather than just looking at the 'collecting from <enterprise address>'. Further, for many enterprises (Baw Baw, all SEFH buying groups, Bondi, Local Organics...), they just have one pick up point, and it is the same as the enterprise address (no conflicting info). Also, for Jonai, her address is '8 hubs around Melbourne' so customers won't accidentally go there- they'll go to the shipping method name ie/Brunswick.

But I've identified a number of enterprise who have >1 pick-up shipping method (Bonnie Beef, 90 Mile, Farm Food Store, Selandra, 80 Acre Farm) . For them, putting 'collecting from <enterprise address>' will conflict with the shipping method description.

@RohanM 
I'm confident the 'collecting from...<enterprise address>' part of the pick-up order confirmation can be deleted.
- I checked all SEFH buying groups and most shops, and if they have a pick up shipping method they either include the address in the description, or it's obvious (ie/ Pick up from Goodstart Early Learning)
- Just in case an enterprise has a pick up shipping method, with no address or description, we could add the enterprise's address to the bottom, under their email address and website. But not essential I don't believe.
